src/main/java/gregtech/api/util/GTUtility.java

```diff
@@ -87,6 +88,14 @@ public static void copyInventoryItems(IItemHandler src, IItemHandlerModifiable d
         }
     }
 
+    public static InventoryCrafting deepCopyInventoryCrafting(final InventoryCrafting source) {
+        final InventoryCrafting result = new InventoryCrafting(new DummyContainer(), source.getWidth(), source.getHeight());
+        for (int i = 0; i < result.getSizeInventory(); ++i) {
+            result.setInventorySlotContents(i, source.getStackInSlot(i).copy());
+        }
+        return result;
+    }
+
```

Comment:
Please move this to new file `InventoryCraftingUtil`

Reply:
If you have a real issue with this code I will fix it. But I am not going to do this.

Reply:
I am trying to keep this file from growing to unreasonable size and having everything mixed up in it (I know that it contains quite a bit of everything now but it should not continue). To make code more readable by separating it to smaller chunks with up to point naming.

I feel quite bad that I have to request these changes as it should be something obvious. But if I don't do it quality of project would go down and it will become even harder to get bearing and start developing on top of it. So please just do it.